File: app/src/main/java/io/github/sithengineer/motoqueiro/ui/statistics/StatisticsComponent.java
package io.github.sithengineer.motoqueiro.ui.statistics;

import dagger.Subcomponent;
import io.github.sithengineer.motoqueiro.scope.ActivityScope;

@ActivityScope @Subcomponent(modules = { StatisticsModule.class })
public interface StatisticsComponent {
  void inject(StatisticsFragment fragment);
}

File: app/src/main/java/io/github/sithengineer/motoqueiro/ui/statistics/StatisticsComponent.kt
package io.github.sithengineer.motoqueiro.ui.statistics

import dagger.Subcomponent
import io.github.sithengineer.motoqueiro.scope.ActivityScope

@ActivityScope
@Subcomponent(modules = [StatisticsModule::class])
interface StatisticsComponent {
  fun inject(fragment: StatisticsFragment)
}
